### ISSUE TITLE
PHP5 Required for Cover production and TVMAZE access

### ISSUE THREAD

**Issue by DrakeJones** [nZEDb/nZEDb#2382]
###Expected behaviour
Fully functional install with php7, no php5 requirments

###Actual behaviour
No covers or TVMAZE access/data without install of some or all of the following: php5-cgi, php5-common, php5-curl (required), php5-gd, php5-jason php5-imagick (required).

###Steps to reproduce the behaviour
Clean install on Debian 8 without php5 components mentioned above.

**Comment by niel** [comment]
This is a distribution or configuration error beyond our control.

**Comment by Bart39** [comment]
As Debian 8 aka Jessie comes with php 5.6 by default (afaik) then I suspect you have not upgraded to 7 properly

All of those packages you mentioned have php7 equivalent versions

https://www.cyberciti.biz/faq/installing-php-7-on-debian-linux-8-jessie-wheezy-using-apt-get/

Has a good example of removing 5.6 and installing 7 correctly on either Jessie / wheezy

**Comment by ghost** [comment]
Upgrading on Ubuntu Xenial also results in some packages not being updated. You will neet to install them manually using apt.

**Comment by DrakeJones** [comment]
Thanks Bart.  I’ll give it a go.

**Comment by DrakeJones** [comment]
Thank you all. I'll follow all the suggestions/recommendations above. Didn't intend to cause any grief or consternation - if the fixes don't fix it, I'll let you know, else - let's call it case closed. 

Addendum - nZEDb is an amazing ecosystem. And, special thanks for your patience and diligence with my issues, Bart.

**Comment by DrakeJones** [comment]
Yup. Have to remove all traces of php5 before upgrading Debian 8 to php7. Removed all php files and dependencies and reinstalled php7...  Removing php-common took out all php7.0 packages. Had to remove and reinstall memcached, igbinaries, etc., with explicit php7 version and that worked. Thank you.

Case closed.

**Comment by Bart39** [comment]
No problem, you didn't cause any grief. You had an issue and we got to the bottom of it

**Comment by DrakeJones** [comment]
You are absolutely correct, Niel.  With Bart’s help, I have resolved the issue.  I uninstalled php* and compiled php7 using this: https://linuxconfig.org/how-to-compile-and-install-php-7-on-debian-jessie-linux.  One minor wrinkle: After upgrading to straight php7, the cli and fpm php.ini’s timezone settings have to be changed one-at-a-time, stopping and restarting tmux in between, or it breaks nZEDb.
